firebase-sessions/test-app/test-app.gradle.kts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id("com.android.application")
+  id("org.jetbrains.kotlin.android")
+  id("com.google.gms.google-services")
+}
+
+android {
+  namespace = "com.google.firebase.testing.sessions"
+  compileSdk = 33
+  defaultConfig {
+    applicationId = "com.google.firebase.testing.sessions"
+    minSdk = 16
+    targetSdk = 33
+    versionCode = 1
+    versionName = "1.0"
+    multiDexEnabled = true
+  }
+  buildTypes {
+    release {
+      isMinifyEnabled = false
+      proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+    }
+  }
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+  kotlinOptions { jvmTarget = "1.8" }
+}
+
+dependencies {
+  implementation(project(":firebase-sessions"))
```

Comment:
Let's add TODO to say that we need to remove this thicker dependency once we have the test app depending on Crashlytics and/or Fireperf.

Something we did in iOS: We have the same app with different flavors - App with Crashlytics and no Fireperf, App with Fireperf and no Crashlytics, App with both. If we have to do something similar, can we do that here as well?

Reply:
Done.